src/applications/coronovirus-screener/components/FormResult.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { questions } from '../config/questions';
+import { Element } from 'react-scroll';
+import moment from 'moment';
+
+export default function FormResult({ formState }) {
+  let result;
+
+  const incomplete = <div>Please answer all the questions above.</div>;
+
+  const pass = (
+    <div>
+      <h2 className="vads-u-font-size--h1">PASS</h2>
+      <h3>Valid for: {moment().format('MMMM Do YYYY, h:mm:ss a')}</h3>
+      <div className="vads-u-font-size--h3">
+        <p>
+          Please show this screen to the staff member at the facility entrance.
+        </p>
+        <p>Thank you for helping us protect you and others during this time.</p>
+      </div>
+    </div>
+  );
+
+  const fail = (
+    <div>
+      <h2 className="vads-u-font-size--h1">More screening needed</h2>
+      <h3>Valid for: {moment().format('MMMM Do YYYY, h:mm:ss a')}</h3>
```

Comment:
Mind dropping the seconds precision here?

Reply:
yes... for all our E2E tests 😁

Reply:
done!